apps/e2e-tests/src/process-compose/e2e.spec.ts

```diff
@@ -50,6 +89,14 @@ describe.sequential('E2E Tests with process-compose', async () => {
     expect(processes).toEqual(expectedPCStatuses03);
   });
 
+  test('Test prices are updated', async () => {
+    const currentPrices = await getPricesInfo(allowFeeds, ADFSConsumer);
```

Comment:
ditto: use `mapValuePromises`
```suggestion
    const currentPrices = await mapValuePromises(
      initialPrices,
      async (feedId, _) =>
        await ADFSConsumer.getLatestSingleData(feedId).then(res =>
          Number(res.slice(0, 50)),
        ),
    );
```